cypress/e2e/view-100/synchronized-view-and-navtree.cy.js

```diff
@@ -46,7 +46,7 @@ describe('View 100: Synchronized View and NavTree', () => {
       waitForModelReady('twoLevelSelect')
     })
 
-    it('Item highlighted in tree and scene - Screen', () => {
+    it('Item highlighted in scene - Screen', () => {
       cy.percySnapshot()
```

Comment:
Please add check that tree is not visible

Reply:
done.

Reply:
done